src/main/java/me/ryanhamshire/GriefPrevention/DatabaseDataStore.java

```diff
@@ -411,6 +411,9 @@ else if (parentId == -1)
             statement.execute("DELETE FROM griefprevention_claimdata WHERE id='-1';");
         }
 
+        results.close();
+        statement.close();
```

Comment:
Good, but there are a couple missed cases earlier in the method where reassignment can leak unclosed resources.

Reply:
did he just mark these all as resolved without doing anything

Reply:
Yes